uint8 IDLE = 1
uint8 INIT = 2
uint8 READY = 3
uint8 ACTIVE = 4
uint8 PAUSED = 5
uint8 STUCK = 6
uint8 STALL = 7
uint8 SLIP = 8
uint8 SHUTDOWN = 9
uint8 RESET = 10
uint8 DONE = 11
int32 status
string text
